firestore-notice-extension/functions/src/converter.ts

```
@@ -0,0 +1,51 @@
+import { FieldValue, FirestoreDataConverter, WithFieldValue } from "firebase-admin/firestore";
+
+import { Notice } from "./interface";
+import { Acknowledgement } from "./interface";
+
+export const noticeConverter: FirestoreDataConverter<Notice> = {
+  toFirestore(notice: any): FirebaseFirestore.DocumentData {
+    throw new Error("Creating a notice is not supported.");
+  },
+  fromFirestore(snapshot: FirebaseFirestore.QueryDocumentSnapshot) {
+    const data = snapshot.data();
+
+    // TODO throw if data is invalid?
```

Should we throw if the data stored on the database is invalid?